app/assets/javascripts/short-form/ShortFormApplicationService.js.coffee.erb

```diff
@@ -309,6 +310,18 @@ ShortFormApplicationService = ($localStorage, $translate, $http, Upload, uuid) -
       Service.preferences["#{fileType}_error"] = true
     ))
 
+  Service.checkSurveyComplete = ->
```

Comment:
I _think_ I know. Is it if a user clicks Edit on the review page and finishing editing the section, they jump straight to the review page? And how we know if they reached the end of the app is if they filled out the optional demographic questions?